pyensembl/sequence_data.py

```diff
@@ -36,22 +37,24 @@ class SequenceData(object):
     """
     def __init__(
             self,
-            fasta_path,
+            fasta_paths,
             require_ensembl_ids=False,
             cache_directory_path=None):
```

Comment:
In the interest of not breaking any other existing code which may use `SequenceData(fasta_path)`, do you mind adding:
```
if isinstance(fasta_paths, six.string_types):
    fasta_paths = [fasta_paths]
```